google/cloud/spanner_dbapi/cursor.py

```diff
@@ -67,6 +67,8 @@ def __init__(self, connection):
         self._row_count = _UNSET_COUNT
         self._connection = connection
         self._is_closed = False
+        # the currently running SQL statement results checksum
+        self._checksum = None
```

Comment:
Fetching is consistent for the concrete statement, so we can use just a pointer to the current statement checksum in cursor.